swarm/tools/runs_gc.py

```
@@ -86,7 +86,9 @@ def get_dir_size(path: Path) -> int:
     return total
 
 
-def get_run_info(run_id: str, run_path: Path, run_type: str) -> RunInfo:
+def get_run_info(
+    run_id: str, run_path: Path, run_type: str, compute_size: bool = True
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)